test/builtin_dataset_mocks.py

```diff
@@ -1390,3 +1390,44 @@ def pcam(info, root, config):
             compressed_file.write(compressed_data)
 
     return num_images
+
+
+@register_mock
+def stanford_cars(info, root, config):
+    import scipy.io as io
+    from numpy.core.records import fromarrays
+
+    num_examples = {"train": 5, "test": 7}[config["split"]]
+    num_classes = 3
```

Comment:
Can we call this

```suggestion
    num_categories = 3
```

to align it with the terminology of the other datasets?